package.json

```diff
@@ -85,7 +85,9 @@
         "@swimlane/ngx-graph": {
             "d3-color": "^3.1.0",
             "d3-interpolate": "^3.0.1",
-            "d3-brush": "^3.0.0"
+            "d3-transition": "^3.0.1",
```

Comment:
is the version for d3-transition on purpose different than the ones for d3-brush and d3-selection?